local-volume/README.md

```diff
@@ -71,7 +71,7 @@ gcloud alpha container node-pools create ... --local-ssd-count=<n>
 2. Mount all the filesystems under one directory per StorageClass. The directories
    are specified in a configmap, see below. By default, the discovery directory is
    `/mnt/disks` and storage class is `local-storage`.
-3. Configure a Kubernetes cluster with the `PersistentLocalVolumes` feature gate.
+3. Configure the Kubernetes API Server with the `PersistentLocalVolumes` & `AffinityInAnnotations` feature gates.
```

Comment:
i couldve sworn that when i setup the api server with only the PersistentLocalVolumes feature gate it didnt work.
i probably didnt properly apply the change, retried it now and it works…
my bad